src/Queue.php

```diff
@@ -118,10 +118,6 @@ public function withAdapter(AdapterInterface $adapter): self
         $new = clone $this;
         $new->adapter = $adapter;
 
-        if ($adapter instanceof QueueDependentInterface) {
```

Comment:
There is no such class. I didn't find when and where he was.

Reply:
@viktorprogger ?